docs/articles/documentation/why-testcafe/README.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: docs
+title: Why TestCafe?
+permalink: /documentation/why-testcafe/
+---
+
+# Why TestCafe?
+
+TestCafe excels where other end-to-end testing solutions fall short.
+
+## One NPM package
+
+TestCafe doesn't require the use of extra tools and frameworks. All you need to get up and running is a recent version of Node.js, a browser — any browser, and the TestCafe NPM package.
+
+## As simple and extendable as JavaScript itself
+
+TestCafe tests are full-fledged Node.js apps. They are compatible with your favourite JavaScript libraries and preprocessors.
```

Comment:
"TestCafe tests are full-fledged Node.js apps... " - Once again I think that it makes TestCafe look more complicated than it really is. Should I create Node.js apps from scratch? Let's make sure that people understand that tests are simple... and internally they may be Node.js apps, but users get the simplicity of tests, yet the capability to use a language / syntax you already know. 

"Compatible with your favorite libraries..." I guess this sentence undermines your previous statement that TestCafe needs to extra frameworks or tools. Here you are talking about extra libraries and preprocessors. 

So if you want to have a message about simplicity, let's define it clearly - what are the boundaries. What can you do with a simple setup and simple things. Extensibility, additional libraries and preprocessors - does that fit into the "simple" narrative? I don't think so. There should be a boundary somewhere between "simple" and "powerful/extendable" and we need to make sure we define it and that the text follows it. 

Here are some thoughts - rough ideas: 

Minimal API, Familiar Syntax
9 out of 10 cases are simple - all you need to to learn TestCafe API... That's less than 20 members that correspond to user action or a few assertions. And the great part is that you write it in JavaScript, which you already know. If you need to access any element, do it as you usually do it. 

Your Website uses Custom Libraries and Preprocessors? Your Tests Can Too!
You're not using vanilla JavaScript on your website? You'd rather have your test utilize the same libraries and pre-processors that you already incorporated into your workflow? Not a problem. TestCafe scripts are Node.js apps internally, so anything that works on your website will work in tests.